Consistent Copyright by the KubeVirt Authors under /pkg

dankenigsberg opened the pull request:
Red Hat [allows/recommends](https://docs.google.com/document/d/17ewwkhXh0NQVgaHZwb1310Wb5XefhzdB7woamC4GR3Y/) simplifying the Copyright message on files. Let us use this simplification under /pkgs so we don't need to think about this boring Legalese under this directory.

PRs for other directories should follow.

/kind cleanup

```release-note
NONE
```

fossedihelm commented on pkg/virt-config/configuration.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm commented on pkg/virt-config/feature-gates.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm commented on pkg/virt-config/virt-config.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm commented on pkg/virt-controller/services/template.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm commented on pkg/virt-controller/watch/application.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm commented on pkg/virt-controller/watch/migration/migration.go:
```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  *
- * Copyright 2017, 2018 Red Hat, Inc.
+ * Copyright 2017, The KubeVirt Authors.
```
same

fossedihelm reviewed (changes requested):
Some leftover dates:
I didn't report them all.
Searching for `, The KubeVirt Authors.` can help.
Thank you

oshoval commented:
Thanks
Sseems there are more combination, some with year, some of IBM / RH / others in case it is interesting please
`git grep "* Copyright" | grep -v  "Copyright The KubeVirt Authors" | grep -v vendor/`

dankenigsberg committed:
Consistent copyright notice under pkg

Red Hat is willing to mark these files as copyrighted by the KubeVirt
Authors. If accepted, a similar sed PR would follow to the reset of the
code based.

sed -i 's/\(20.., \)*20.. Red Hat, Inc./The KubeVirt Authors./' `git grep -l '20.. Red Hat, Inc.'`

Signed-off-by: Dan Kenigsberg <danken@redhat.com>

avlitman commented:
/test pull-kubevirt-unit-test

avlitman commented:
@dankenigsberg 

you can run this .sh script:

```
#!/usr/bin/env bash
set -euo pipefail
set -x

ROOT_DIR="$(git rev-parse --show-toplevel)"
echo "Repo root: $ROOT_DIR"

files_to_update=$(git grep -l "Copyright" "$ROOT_DIR" | grep -v "Copyright The KubeVirt Authors" | grep -v "vendor/")

for file in $files_to_update; do
    echo "Updating copyright in: $file"
    sed -i 's/Copyright .*/Copyright The KubeVirt Authors./' "$file"
done

```

dankenigsberg commented:
@avlitman we got permission/suggestion to modify Red Hat's Copyright notice, not anyone else's.

0xFelix reviewed (commented):
Some files in `kubevirt/kubevirt` use `The KubeVirt Authors` while others use `The KubeVirt Authors.`. What's better? This is a super nit though.

dankenigsberg committed:
consistent punctuation of Copyright notices under pkg/

Signed-off-by: Dan Kenigsberg <danken@redhat.com>

dankenigsberg commented:
> Some files in `kubevirt/kubevirt` use `The KubeVirt Authors` while others use `The KubeVirt Authors.`. What's better? This is a super nit though.

No idea. But I've added the period for consistency.

oshoval commented:
> @avlitman we got permission/suggestion to modify Red Hat's Copyright notice, not anyone else's.

Nice to have - open source infringement linter (i.e in this case that makes sure copyright isnt changed unless the combination is whitelisted), maybe something like that exists, if desired will check in the future once i have time

avlitman commented:
/lgtm

ShellyKa13 commented:
I think there is no need for two commits, they should be squashed to one but obviously not a blocker :)

enp0s3 commented:
/approve
@dankenigsberg Thank you for the effort.

dankenigsberg commented:
> I think there is no need for two commits, they should be squashed to one but obviously not a blocker :)

You are right. Sorry for not having them squashed.
